packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -241,13 +241,98 @@ Should only generate one model named User.
 
 Expected input body:
 
+```json
+{
+  "name": "Madge"
+}
+```
+
+Expected response body:
+
+```json
+{
+  "id": 1,
+  "name": "Madge"
+}
+```
+
+### Azure_Core_createOrReplace
+
+- Endpoint: `get /azure/core`
```

Comment:
We might need some improvement on summary util. I assume the "get" is default when cadl source does not have e.g. `@put`. But for Azure.Core, these are in maybe `@createsOrReplacesResource(TResource)`.